notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Adjuste the scores
</pre><p>Adjust?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Good catch! Fixed.